go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -27,6 +28,8 @@ func main() {
 	master := flag.String("init-master", "", "initialize master, requires the hostname as argument")
 	node := flag.String("init-node", "", "initialize node, requires the name that node is registered with in kubernetes cluster")
 
+	logrus.SetLevel(logrus.DebugLevel)
+
```

Comment:
We should probably get the input from the user to set the log level. I tried doing this in my own patch and ended up like:

```
+       // log flags
+       verbose := flag.Int("loglevel", 5, "logrus loglevels")
+
        flag.Parse()

+       // Process log flags
+       logrus.SetLevel(logrus.Level(*verbose))
+

```
But, we still get the -V option, which IMO, we should do something about.

Reply:
Yes. I was going to fix this as an option. Will roll out a new PR for this.